src/actions/ui.js

```diff
@@ -170,7 +170,7 @@ export function clearProjectDirectoryRoot() {
 export function setProjectDirectoryRoot(newRoot: string) {
   return ({ dispatch, getState }: ThunkArgs) => {
     const curRoot = getProjectDirectoryRoot(getState());
-    if (newRoot && curRoot) {
+    if (newRoot || curRoot) {
```

Comment:
`|| curRoot` doesn't really do anything here.
1. What is it's intended purpose? Why are we not using it? Is it not needed?